Documentation/reference/f5-ltm-lb.md

```diff
@@ -0,0 +1,92 @@
+# Using F5 BIG-IP LTM with Tectonic
+
+Installing Tectonic on bare metal or VMWare requires a master DNS (which resolves to any master node in the cluster), and a worker DNS (which resolves to any worker nodes for Ingress). Evaluation clusters may be set up using a DNS entry that points to each node, but production clusters require the use of a load balancer.
+
+When installing on cloud providers such as AWS, Tectonic automatically creates a load balancer (such as AWS ELB). When installing Tectonic on bare metal or VMWare, you must provide your own load balancer.
```

Comment:
s/(such as AWS ELB)/, such as an ELB on AWS/

(consider linking to AWS doc on ELBs)